Documentation/CRDs/ceph-cluster-crd.md

```diff
@@ -460,7 +460,7 @@ This feature is only available when `useAllNodes` has been set to `false`.
 
 Below are the settings for host-based cluster. This type of cluster can specify devices for OSDs, both at the cluster and individual node level, for selecting which storage resources will be included in the cluster.
 
-* `useAllDevices`: `true` or `false`, indicating whether all devices found on nodes in the cluster should be automatically consumed by OSDs. **Not recommended** unless you have a very controlled environment where you will not risk formatting of devices with existing data. When `true`, all devices/partitions will be used. Is overridden by `deviceFilter` if specified.
+* `useAllDevices`: `true` or `false`, indicating whether all devices found on nodes in the cluster should be automatically consumed by OSDs. **Not recommended** unless you have a very controlled environment where you will not risk formatting of devices with existing data. When `true`, all devices, partitions, and logical volumes will be used. Is overridden by `deviceFilter` if specified.
```

Comment:
Do we really want to consume all LVs automatically with `useAllDevices: true`? I had a similar concern with partitions and it was actually ok, but maybe for LVs it would be a good extra caution to require matching to the deviceFilter or specific device path name. But maybe that is going to complicate this change too much. Wdyt?

Reply:
It would be nice to force specifying LV names because `useAllDevices: true` is not recommended and I want to avoid a regression caused by this new feature.

I'll try to add this limitation. Please wait for a while.